Documentation/templates/ib/scenes-swift3.md

```diff
@@ -2,24 +2,23 @@
 
 | Name      | Description       |
 | --------- | ----------------- |
-| File name | ib/swift3.stencil |
-| Invocation example | `swiftgen ib -t swift3 …` |
+| File name | ib/scenes-swift3.stencil |
+| Invocation example | `swiftgen ib -t scenes-swift3 …` |
 | Language | Swift 3 |
 | Author | Olivier Halligon |
 
 ## When to use it
 
-- When you need to generate *Swift 3* code
-- The generated code supports both UIKit platforms (iOS, tvOS and watchOS) and AppKit platform (macOS)
+- When you need to generate *Swift 3* code for your storyboard *scenes*.
+- The generated code supports both UIKit platforms (iOS, tvOS and watchOS) and AppKit platform (macOS).
```

Comment:
Might be interesting to add a 3rd bullet point just informing that: "Note: if you also need to generate code for your storyboard segues, you can use [that other template] in addition to this one" ?

Reply:
Good idea, will do.